### DISCUSSION TITLE
Help setting up with homeassistant

### DISCUSSION THREAD

**Discussion by syrusfem** [syssi/esphome-atorch-dl24#62] [Q&A]
Hi,
 i have got an DT24PW and i am trying to integrate it in Homeassistant via EspHome but the instruction page is a bit confusing for me, and i am only beginning with EspHome .
The code of the yalm described in the instruction is to be inserted in the YALM of an ESP32 configured in the ESPHome add on of Homeassistant or i must install and edit Yalm file aoutside of homeasssitant?
Sorry if i am generic in my reuqest but i am a  bit lost of what to do.

**Comment by syssi** [comment]
Did you succeed or do you still need some support?

**Reply by syrusfem** [reply]
hi, i had a bad DT24PW (no power on) and returned it, so when i'll have the new board i think i'll try the german guide in discussion (that i did't noticed first time) . So for the moment my project of integration in in stand by.

**Reply by Phobos-7** [reply]
did it ever work for you..?

**Reply by syrusfem** [reply]
No, can't do, maybe it is my skill , maybe some change of the Firmware. I switched to a GC97  https://community.home-assistant.io/t/esphome-solar-system-any-suggestions-for-tk15-coulomb-meter-battery-monitor-integration/323930/5  https://github.com/alidonet/GC97_ESP it is a bit more involved but i set 2 of these on two systems and they work good with HA.

**Reply by syssi** [reply]
@Phobos-7 Do you need some support?
